manifests/config.pp

```diff
@@ -156,25 +156,21 @@
     # }
 
     # Generate Elasticsearch config
-    $_es_config = merge(
+    $data = merge(
       $elasticsearch::config,
       { 'path.data' => $elasticsearch::datadir },
       { 'path.logs' => $elasticsearch::logdir },
       $_tls_config
     )
 
-    datacat_fragment { 'main_config':
```

Comment:
any idea why this was used in the first place? To concatenate multiple strings into one file?

Reply:
I suppose it was to merge 1 or more data in one file (with older puppet version)